src/ffs/index.ts

```diff
@@ -8,13 +8,60 @@ import { promise } from "../util"
 import { ListDealRecordsOption, PushStorageConfigOption, WatchLogsOption } from "./options"
 import { coldObjToMessage, hotObjToMessage } from "./util"
 
+export interface FFS {
```

Comment:
so i ran into this issue of wanting a typed method to simplify the docs. for example, 

instead of having to reinitialize an instace of `pow` in every doc string, i just wrap them in functions that take an input of the `pow` api object. however, there was no type for that object. so it would have been like,

`function(pow: any) => things`

where i wanted

`function (pow: POW) => stuff`

that nice, single type for the whole api didn't exist. so here i created it. not sure how else to achieve this goal of typing the api.